scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -486,14 +511,18 @@ def publish_release(new_release, take_action):
     # Create draft release with message that includes download URLs and history
     try:
         github_release_create(release_name, notes)
+    except urllib.error.URLError as e:
+        print(f"Failed to create the release {release_name}: {e}")
+        print(f"Response: {str(e.fp.readlines())}")
```

Comment:
Should we add response printout the to catch-all handler?

Reply:
I think it's a different type so it doesn't have these fields

Reply:
Can you explain when this exception is thrown?

Reply:
For me it when I tested and the release was already present. Original log would only say: `Failed to create the release 4.16.0-ec.1-202401101824.p0: HTTP Error 422: Unprocessable Entity` and the hint that there already was a release was in the response body of the HTTP request.